README.md

```diff
@@ -13,14 +13,19 @@ through the docker container found under the docker directory.
 
 Running the backend requires docker. Install ```Docker Desktop``` for windows and ```docker``` on unix.
 
-Building the docker container that hosts the predefined MariaDB:
-```
-cd <project root>/docker
-docker build -t mariadb:<tag number> .
+### Setup
+
+You can set up the backend with the following command. It will create a `.env` file, a docker image of mariadb:lts and install all npm packages that you will need for the project.
+```sh
+npm run setup
 ```
+### Run the project
+
+In order to run the backend you need to start your created docker container and the backend itself. 
+It is necessary that the docker container is running first.
 
 Running the container (anywhere):
-```
+```sh 
 docker run -p 3306:3306 mariadb:<tag number>
```

Comment:
What would the tag be here? Previously you would run `docker build` manually and set the docker tag with for example `-t mariadb:1`. What is the default for this now?